cobrawap/pipeline/stage03_trigger_detection/Snakefile

```diff
@@ -102,6 +102,7 @@ use rule template as minima with:
                'plot_channels', 'plot_tstart', 'plot_tstop',
                 img_name='minima_channel0.'+config.PLOT_FORMAT, config=config)
 
+
```

Comment:
Comment headings are not preceded by two blank lines.

Reply:
We should instead use two blank lines before comment headings, so to better distinguish rules belonging to different "groups", given that rules from the same group are separated by double blank lines anyway. Let's decide a norm, and then let's stick to it.